src/components/Header.js

```diff
@@ -1,19 +1,33 @@
-import React, { Component } from 'react';
-import Logo from '../assets/turtle_header.jpeg'
-class Header extends Component {
-    render() {
+// import React, { Component } from 'react';
+import React, { useState } from 'react';
+import { Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';
+// import Logo from '../assets/turtle_header.jpeg'
```

Comment:
are you saving this for later implementation?

Reply:
took care of these. we can add them later if we need them.